styles/SASS/_pokemon-choose-view.scss

```diff
@@ -1 +1,52 @@
-// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
\ No newline at end of file
+// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
+
+#choose-page {
+    width: 100%;
+    height: 100%;
```

Comment:
height needs be change to 100vh instead of 100% otherwise content dont fill all view height
![image](https://user-images.githubusercontent.com/74777578/106804344-92780000-6665-11eb-8a46-01d382925f8d.png)